documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -149,6 +150,14 @@ if the Jenkins servers were down and the payload was not received.
 Using webhooks is a more efficient method to automatically trigger builds than
 polling, as it does not require Jenkins to query GitHub on regular intervals.
 
+### GitHub PR Labels
+
+GitHub PRs are also pass labels through to the Jenkins build system. These labels can be accessed via the `PR_LABELS` environment variable from within scripts. Currently, there are 3 labels of importance to the Jenkins system.
+
+- `DO_NOT_CI` - Label which skips the Jenkins system entirely (primarily for use with documentation changes).
+- `do-not-build` - Label which skips the build and test phases of the pipeline (primarily for use with Jenkins Pipeline changes).
```

Comment:
What will the CI do if it is active but not building or testing?

Reply:
The other stages, which are labelled in the Jenkins. i.e. Analyse, build docs, package, etc.

Reply:
So do not build means do not build code, not do not build anything. May be ambiguous?

Reply:
It is "skip the build stage (and thereby the test stage) of the pipeline"

Reply:
Just thinking that "... (but doesn't skip the build docs ..." could make things more explicit

Reply:
My concern is that then you get into an arms race about what it doesn't skip. I might change the word "phase" to "stage" to align with Jenkins terminology if you'd prefer.

Reply:
Sounds as if you feel strongly about this. My aim is to disambiguate the word build, which is  pointing at what gets produced. Changing phase to stage still requires the reader to do some Jenkins interpretation rather than just noting "builds this, doesn't build that". Whatever text gets that across will be fine by me.